doc/contribution-guidelines.md

```diff
@@ -0,0 +1,97 @@
+# Contribution guidelines
+
+- [Pull Requests](#pullrequest)
+- [Commit Message Guidelines](#commit)
+
+## <a name="pullrequest"></a> Pull Requests Guidelines
+Pull requests are the only means by which you can contribute to this project, please follow the following steps when submitting pull requests :
+
+1. Create your pull request. (PR name must match [commit name](#commit))
+2. Fill the pull request template with all of the required elements
+3. CI tests kick in and report back any issues, you should fix these issues before continuing. No broken PR will be reviewed.
```

Comment:
I agree with you, these were inspired from private work where developers have sufficient knowledge to debug failing PRs. We can get rid of it